frontend/src/metabase/components/FieldValuesWidget/FieldValuesWidget.tsx

```diff
@@ -432,9 +443,9 @@ export function FieldValuesWidgetInner({
       <div
         data-testid="field-values-widget"
         style={{
-          width: expand ? maxWidth : containerWidth,
-          minWidth: minWidth,
-          maxWidth: maxWidth,
+          width: (isExpanded ? maxWidth : containerWidth) ?? undefined,
```

Comment:
does it hurt if you pass `null`? (most likely you're covering this edge case with `??`

Reply:
Heah, types are defined like `width?: number` so it's not happy with nulls 🥴